app/views/timelog/_date_range.html.erb

```diff
@@ -26,40 +26,35 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
-<fieldset id="date-range" class="collapsible">
-  <legend onclick="toggleFieldset(this);"><a href="javascript:"><%= l(:label_date_range) %></a></legend>
-  <div>
-    <div class="form--field">
-      <%= styled_label_tag "period_type_list", l(:description_date_range_list), :class => "hidden-for-sighted" %>
+<fieldset id="date-range" class="simple-filters--container">
+  <legend><%= l(:label_date_range) %></legend>
+  <ul class="simple-filters--filters">
+    <li class="simple-filters--filter">
+      <%= styled_label_tag "period_type_list", l(:description_date_range_list), :class => "hidden-for-sighted simple-filters--filter-name" %>
       <%= styled_radio_button_tag 'period_type', '1',
                                   !@free_period,
-                                  :onclick => 'Form.Element.disable("from");Form.Element.disable("to");Form.Element.enable("period");',
                                   :id => "period_type_list"%>
-      <%= styled_select_tag 'period',
-                            options_for_period_select(params[:period]),
-                            :onfocus => '$("period_type_1").checked = true;',
-                            :disabled => @free_period,
-                            class: "-narrow" %>
-    </div>
-    <div class="grid-block small-6">
-      <%= styled_label_tag "period_type_interval", l(:description_date_range_interval), :class => "hidden-for-sighted" %>
-      <%= styled_radio_button_tag 'period_type', '2', @free_period, :onclick => 'Form.Element.enable("from");Form.Element.enable("to");Form.Element.disable("period");', :id => "period_type_interval" %>
-
-      <span onclick="$('period_type_2').checked = true;" class="grid-block">
-      <div class="form--field">
-        <%= styled_label_tag("from", l(:label_date_from)) %>
-        <%= styled_text_field_tag('from', @from, :size => 10, :disabled => !@free_period) %>
+      <div class="simple-filters--filter-value">
+        <%= styled_select_tag 'period',
+                              options_for_period_select(params[:period]),
+                              class: "-narrow" %>
+      </div>
+    </li>
+    <li class="simple-filters--filter">
+      <%= styled_label_tag "period_type_interval", l(:description_date_range_interval), :class => "hidden-for-sighted simple-filters--filter-name" %>
+      <%= styled_radio_button_tag 'period_type', '2', @free_period, :id => "period_type_interval" %>
+      <%= styled_label_tag("from", l(:label_date_from), class: 'simple-filters--filter-name') %>
+      <div class="simple-filters--fitler-value">
```

Comment:
I also always mis-type `filter` for `fitler`. Still wrong, though:

~~`simple-filters--fitler-value`~~
`simple-filters--filter-value`